src/Libraries/CoreNodes/Properties/Resources.Designer.cs

```diff
@@ -158,5 +158,14 @@ internal class Resources {
                 return ResourceManager.GetString("QuadtreeConstructionNullUVSetMessage", resourceCulture);
             }
         }
+        
+        /// <summary>
+        ///   Looks up a localized string similar to Use File.FromPath -&gt; Image.ReadFromFile -&gt; Image.Pixels nodes instead..
```

Comment:
Then I guess we can remove the "." in the string to be formatted. Right? Although this is pretty minor.